cluster/images/hyperkube/static-pods/kube-proxy.json

```diff
@@ -0,0 +1,27 @@
+{
+  "apiVersion": "v1",
+  "kind": "Pod",
+  "metadata": {
+    "name": "k8s-proxy",
+    "namespace": "kube-system"
+  },
+  "spec": {
+    "hostNetwork": true,
+    "containers": [
+      {
+        "name": "kube-proxy",
+        "image": "gcr.io/google_containers/hyperkube-ARCH:VERSION",
+        "command": [
+                "/hyperkube",
+                "proxy",
+                "--master=http://127.0.0.1:8080",
```

Comment:
Yep, I'd like to fix everything up soon (with proper tools like ConfigMap and ComponentConfig and so on).
Static pods are only run on the master, so it works. On workers kube-proxy is run manually, and the `--master` flag is set by the bash script

I'll fix it up now on the way to v1.4, but we should discuss about a general solution that is configurable and works for everyone, and move to that solution... I know there are lots of threads about that already.